Graphs/Components/utils/GraphAxis.js

```diff
@@ -22,11 +23,14 @@ const GraphAxis = (props) => {
     dx = dx || 0;
     dy = dy || 0;
 
-    const parsedData = Formatter({
-        dateHistogram,
-        value: payload.value,
-        tickFormat
-    })
+    const parsedData = tickLabel  && tickLabel[payload.value] ? 
```

Comment:
Let's make this user configurable. Introduce a property called `xTicksLabelFormatted`. Set the default value to `true` in `GraphAxis` when we are using the `Formatter`. When the user sets it to `false` in `vis-config`, you will just display the label using `tickLabel[payload.value]`. This was we are always using `Formatter` by default.

Reply:
This is already user configurable based on ticklabel is sent or not. But as per our discussion moved translation into formatter to keep all transformations in one place